packages/hardhat-core/sample-projects/advanced-ts/.eslintrc.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  env: {
+    browser: false,
+    es2021: true,
+    mocha: true,
+    node: true,
+  },
+  plugins: ["@typescript-eslint"],
+  extends: [
+    "standard",
+    "plugin:prettier/recommended",
+    "plugin:node/recommended",
+  ],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaVersion: 12,
+  },
+  overrides: [
+    {
+      files: ["scripts/**"],
+      rules: { "no-process-exit": "off" },
+    },
```

Comment:
We can remove this if we change the sample script's `main` invocation to this:

```ts
main().catch((error) => {
  console.error(error);
  process.exitCode = 1;
});
```

This also apply to the js version

Reply:
Addressed in https://github.com/nomiclabs/hardhat/pull/1770/commits/d88015ccac5f26737b42ee7be5cd42e020cb3b30